CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### citus-docker v9.2.2.docker (March 6, 2020) ###
+
+* Bump Citus version to 9.2.2
```

Comment:
add a new entry maybe?
`* Bump PostgreSQL version to 12.2`

Reply:
thanks :)